server/plugin/command.go

```diff
@@ -600,6 +600,10 @@ func (p *Plugin) handleUnsubscribe(_ *plugin.Context, args *model.CommandArgs, p
 	owner = strings.ToLower(owner)
 	repo = strings.ToLower(repo)
 	if err := p.Unsubscribe(args.ChannelId, repo, owner); err != nil {
+		if strings.Contains(err.Error(), "no subscription exists") {
+			return err.Error()
+		}
```

Comment:
I don't like the implied checks for errors. Can we use a dedicated error type and check that instead?